src/interfaces/store-interfaces/state-model-interface.ts

```diff
@@ -8,18 +8,26 @@ import {
 export interface StateModelIF {
   tombstone: TombStoneIF
   accountInformation: AccountInformationIF
-  businessInformation: BusinessInformationIF,
+  businessInformation: BusinessInformationIF
   nameRequest: NameRequestIF
   nameTranslations: NameTranslationIF | NameTranslationDraftIF[]
   certifyState: CertifyIF
   defineCompanyStep: DefineCompanyIF
-  peopleAndRoles: PeopleAndRolesIF
-  createShareStructureStep: ShareStructureIF
+  peopleAndRolesStep: PeopleAndRolesIF
+  shareStructureStep: ShareStructureIF
   incorporationAgreementStep: IncorporationAgreementIF
-  incorporationDateTime: DateTimeIF,
-  originalIA: IncorporationFilingIF,
+  incorporationDateTime: DateTimeIF
+  originalIA: IncorporationFilingIF
   originalAlteration: AlterationFilingIF
-  staffPaymentStep: StaffPaymentStepIF,
-  filingData: FilingDataIF,
+  staffPaymentStep: StaffPaymentStepIF
+  filingData: FilingDataIF
   detail: DetailIF
+  editingFlags: {
```

Comment:
The new flags, for near-future use.

What do you think, should I do the same thing for the valid and changed flags?